src/udev/udevadm-trigger.c

```diff
@@ -48,6 +49,19 @@ static int exec_list(
                         continue;
                 }
 
+                if (arg_database_match >= 0) {
+                        r = sd_device_get_is_initialized(d);
+
```

Comment:
please keep function call and immediate error handling together, i.e. drop this empty line.

we like empty lines between the error handling of one call and the beginning of the next but please no unnecessary whitespace between calls and their immediate error handling